src/asr/asr_chainer.py

```diff
@@ -96,12 +95,11 @@ def evaluate(self):
 
 
 class ChainerSeqUpdaterKaldi(training.StandardUpdater):
```

Comment:
Same comment. Now you don't have to pass "reader" and you can use built-in `chainer.training.updaters.StandardUpdater` instead of making our own Updater.

Reply:
Now, I am checking "chainer.training.updaters.StandardUpdater" code.
It seems almost same as "ChainerSeqUpdaterKaldi" as you said.
As far as I checked, the differences are the following: (1) check whether grad norm is nan and (2) delete_feat.
Is it OK to use "chainer.training.extensions.Evaluator"?